resources/assets/components/Users/UserInformation.js

```diff
@@ -43,53 +53,36 @@ const UserName = ({ user, link }) => {
   return <span>{displayName}</span>;
 };
 
-const UserInformation = props => (
+export default ({ user, linkSignup, children }) => (
   <div>
-    {!isEmpty(props.user) ? (
+    {!isEmpty(user) ? (
       <div className="container -padded">
         <h2 className="heading">
           <UserName
-            user={props.user}
-            link={props.linkSignup ? `/signups/${props.linkSignup}` : null}
+            user={user}
+            link={linkSignup ? `/signups/${linkSignup}` : null}
           />
         </h2>
         <p>
-          {props.user.email_preview ? (
+          {user.emailPreview ? (
             <span>
-              {props.user.email_preview}
+              {user.emailPreview}
               <br />
             </span>
           ) : null}
 
-          {props.user.mobile_preview ? (
+          {user.mobilePreview ? (
             <span>
-              {props.user.mobile_preview}
+              {user.mobilePreview}
               <br />
             </span>
           ) : null}
 
-          <UserLocation user={props.user} />
+          <UserLocation user={user} />
         </p>
       </div>
     ) : null}
 
-    {props.children}
+    {children}
   </div>
 );
-
-UserInformation.propTypes = {
```

Comment:
Are we cool without the prop type checking?

Reply:
I haven't found a lot of value from prop-type checks since they don't run as static analysis on the pull request (and so I've often missed them if I'm not careful). I also think that as we move towards GraphQL queries, we get the benefits of a typed response out of the box (so re-typing with prop-types seems a little redundant).

Reply:
Ohhh right, that's interesting re GraphQL! I wonder if there's a way to like actually run static analysis on the PR so it's more useful for non-gql props, and then we could have a healthy balance of type checking where appropriate (with maybe more lenient lint rules or some concession for gql props 🤔)?

Reply:
I don't think we can with prop-types (since JavaScript is fairly lenient and so there's often no way for something like ESLint to know what types would be passed to a component until the app runs). But I have [some ideas](https://www.typescriptlang.org) on how we might do more static analysis in the future…